deploy/deploy/crane-manager/deployment.yaml

```diff
@@ -35,8 +35,8 @@ spec:
           operator: Exists
       containers:
```

Comment:
I think keep the folder structor now is better,  this is duplicated `deploy/deploy/crane-manager/deployment.yaml `